bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obs/plannedtime/PlannedTime.java

```diff
@@ -57,7 +57,7 @@ public enum Category {
         CONFIG_CHANGE("Config Change"),
         EXPOSURE("Exposure"),
         READOUT("Readout"),
-        DHS_WRITE("DHS Write"),
+        WRITE("Write"),
```

Comment:
I think this may break the tests which expect the output web page to say "DHS Write"

Reply:
@framos-gemini - if you want to keep the new `WRITE` I think it would be sufficient to use:
WRITE("DHS Write")

Reply:
Hi @andrewwstephens, when I made the new commit I hadn't seen your comment. Thank you very much for that. 

I made the change so that the Observation Overheads table would show Write instead of "DHS Write" because Ghost uses GDS and not DHS. This change produced doubts in Shane and Carlos because it affects to the serialization (they could give more details). 

Also since the deadline for the Ghost ITC is next week and I have other urgent tasks. I prefer to go back, and when I have more time to make the change with the "Ghost Test" implementation. 

<img width="337" alt="Screen Shot 2023-03-31 at 20 50 17" src="https://user-images.githubusercontent.com/78615441/229251487-d90c89ca-0be1-45d8-8a8d-e00eaf9c9cc1.png">

Reply:
:+1: